package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ts-ledger",
```

Comment:
I would really prefer if we can just stick to TypeScript since the team is already ramped up and used to it because of CRUX. 
Any particular reason for not using Typescript code? Name says ts-ledger.